### PR TITLE
Matic word replaced with Polygon (Matic) in DH Docs (Please review carefully)

### PR DESCRIPTION
**Pull request by yash-sharma1**
This PR contains changes in DH Docs where repetition of the word "Matic" replaced by "Polygon (Matic)" These changes majorly are in headings, sub-headings, sentences, still please review thoroughly to avoid replacement of words at an incorrect place like in code/library, etc.

### CODE REVIEW

File: network-documentation/matic/tutorials/untitled.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 ## What this tutorial covers
 
-This tutorial is a brief introduction on how to transfer tokens between Ethereum and Matic on PoS bridge using _**matic.js SDK and Metamask**_. Matic-Ethereum bridge provides a cross-chain channel using which users can transfer tokens from Ethereum to Matic and vice-versa. More details on using the bridge can be found [here](https://docs.matic.network/docs/develop/ethereum-matic/pos/getting-started). This **tutorial mainly focuses on using the bridge from a front end perspective**. We will be using Metamask for this purpose.
+This tutorial is a brief introduction on how to transfer tokens between Ethereum and Polygon (Matic) on PoS bridge using _**matic.js SDK and Metamask**_. Polygon (Matic)-Ethereum bridge provides a cross-chain channel using which users can transfer tokens from Ethereum to Polygon (Matic) and vice-versa. More details on using the bridge can be found [here](https://docs.matic.network/docs/develop/ethereum-matic/pos/getting-started). This **tutorial mainly focuses on using the bridge from a front end perspective**. We will be using Metamask for this purpose.
```
Comment (pyr0gan) [inline network-documentation/matic/tutorials/untitled.md:11]: > This tutorial is a brief introduction on how to transfer tokens between Ethereum and Polygon (formerly Matic Network) on PoS bridge using _**matic.js SDK and Metamask**_. The Polygon-Ethereum bridge provides a cross-chain channel through which users can transfer tokens from Ethereum to Polygon and vice-versa. More details on using the bridge can be found [here](https://docs.matic.network/docs/develop/ethereum-matic/pos/getting-started). **This tutorial mainly focuses on using the bridge from a front end perspective**. We will be using Metamask for this purpose.

File: network-documentation/matic/tutorials/untitled.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-The most important thing to be understood from this tutorial is the **proper usage of the web3 provider in the matic.js instance** we create. Whether using PoS or Plasma, certain actions need to be performed on Matic and some on Ethereum. Due to this reason, **different providers are required in different scenarios. Hence correctly setting the providers is very necessary.**
+The most important thing to be understood from this tutorial is the **proper usage of the web3 provider in the matic.js instance** we create. Whether using PoS or Plasma, certain actions need to be performed on Polygon (Matic) and some on Ethereum. Due to this reason, **different providers are required in different scenarios. Hence correctly setting the providers is very necessary.**
```
Comment (pyr0gan) [inline network-documentation/matic/tutorials/untitled.md:13]: > The most important thing to be understood from this tutorial is the **proper usage of the web3 provider in the matic.js instance** we create. Whether using PoS or Plasma, certain actions need to be performed on Polygon and some on Ethereum. Due to this reason, **different providers are required in different scenarios. Hence correctly setting the providers is very necessary.**

File: network-documentation/matic/tutorials/untitled.md [hunk 3]
```diff
@@ -64,13 +64,13 @@ During deposit of ERC20 tokens, the providers are specified as below
...
+> NOTE: Deposits from Ethereum to Polygon (Matic) happen using a state sync mechanism and takes about ~5-7 minutes. After waiting for this time interval, it is recommended to check the balance using web3.js/matic.js library or using Metamask. The explorer will show the balance only if at least one asset transfer has happened on the child chain. This [link](https://docs.matic.network/docs/develop/ethereum-matic/pos/deposit-withdraw-event-pos/) explains how to track the deposit events.
 
 ![](https://docs.matic.network/img/pos-using-metamask/deposit.png)
 
 ### Transfer
 
-Once deposited, the token can be transferred to any other account on the Matic chain.
+Once deposited, the token can be transferred to any other account on the Polygon (Matic) chain.
```
Comment (pyr0gan) [inline network-documentation/matic/tutorials/untitled.md:73]: Once deposited, the token can be transferred to any other account on the Polygon chain.

File: network-documentation/matic/tutorials/untitled.md [hunk 4]
```diff
@@ -123,5 +123,5 @@ await maticPoSClient.exitERC20(burnTxHash, {
 
 ![](https://docs.matic.network/img/pos-using-metamask/exit.png)
 
-If you had any difficulties following this tutorial or simply want to discuss Matic and DataHub tech with us you can [join our community](https://discord.gg/Chhuv5zHy3) today!
+If you had any difficulties following this tutorial or simply want to discuss Polygon (Matic) and DataHub tech with us you can [join our community](https://discord.gg/Chhuv5zHy3) today!
```
Comment (pyr0gan) [inline network-documentation/matic/tutorials/untitled.md:126]: > If you had any difficulties following this tutorial or simply want to discuss Polygon (formerly Matic Network) or DataHub tech with us, you can [join our community](https://discord.gg/Chhuv5zHy3) today!

File: network-documentation/network-guide.md [hunk 5]
```diff
@@ -42,9 +42,9 @@ Filecoin is a peer-to-peer network that stores files, with built-in economic inc
 
 {% page-ref page="filecoin/" %}
 
-## ![](../.gitbook/assets/matic_network.png) Matic
+## ![](../.gitbook/assets/matic_network.png) Polygon (Matic)
 
-Matic Network is a sidechain-based scaling solution for public blockchains. It is based on an adapted implementation of Plasma framework. Matic provides scalability while ensuring a superior user experience in a secured and decentralized manner. It has a working implementation for Ethereum. Matic intends to support other blockchains in the future which will enable it to provide interoperability features alongside offering scalability to existing public blockchains.
+Polygon (Matic) Network is a sidechain-based scaling solution for public blockchains. It is based on an adapted implementation of Plasma framework. Polygon (Matic) provides scalability while ensuring a superior user experience in a secured and decentralized manner. It has a working implementation for Ethereum. Polygon (Matic) intends to support other blockchains in the future which will enable it to provide interoperability features alongside offering scalability to existing public blockchains.
```
Comment (pyr0gan) [inline network-documentation/network-guide.md:47]: > Polygon (formerly Matic Network) is a sidechain-based scaling solution for public blockchains. It is based on an adapted implementation of Plasma framework. Polygon provides scalability while ensuring a superior user experience in a secured and decentralized manner. It has a working implementation for Ethereum. Polygon intends to support other blockchains in the future which will enable it to provide interoperability features alongside offering scalability to existing public blockchains.

**Review by pyr0gan — CHANGES_REQUESTED** [review]
Take a look, let me know what you think. At least from my perspective, reading Polygon (Matic) _every_ time is a bit much. I tried to make it sensible ⭐

**Comment by yash-sharma1** [comment]
@pyr0gan actually we just need to replace "Matic" exactly with "Polygon (Matic)" as per their team asked.

**Pull request merged**